dao/src/main/java/greencity/repository/EcoNewsRepo.java

```diff
@@ -54,6 +55,16 @@
      */
     Page<EcoNews> findAllByOrderByCreationDateDesc(Pageable page);
 
+    /**
+     * Method that finds {@link EcoNews} by id.
+     * 
+     * @param id {@link Long}.
+     * @return {@link Optional} of {@link EcoNews}
+     *
+     */
+    @Query("select e from EcoNews e join fetch e.tags where e.id = :id")
```

Comment:
It is a good approach to have similar code style to all queries. I would prefer UPPER CASE for key words as SELECT, WHERE etc.